### ISSUE TITLE
baritone 1.16.4 fabric and 1.17 fabric

### ISSUE THREAD

**Issue by eXpertdUst19** [cabaletta/baritone#2819]
please support 1.16.4 fabric and 1.17 fabric

**Comment by wagyourtail** [comment]
1.16.5 build works on 1.16.4, and 1.17 should work when baritone gets updated, waiting on forge toolchain to update.

also duplicate of #2425
